src/pages/LoginPage.tsx

```diff
@@ -1,21 +1,43 @@
-import { Box, Button, Checkbox, Flex, FormControl, FormLabel, Heading, Input, Link, Stack } from '@chakra-ui/react';
-import { Link as LinkRouter } from 'react-router-dom';
+import { Alert, AlertIcon, Box, Button, Checkbox, Flex, FormControl, FormLabel, Heading, Input, Link, Stack } from '@chakra-ui/react';
+import { Link as LinkRouter, useNavigate } from 'react-router-dom';
 import { Formik, Form, Field, ErrorMessage } from 'formik';
 import * as Yup from 'yup';
 import { LoginFormType } from '@/typings/form.type';
+import { authApi, initializeApis } from '../api/api';
+import { accessExpired, setAccessToken } from '@/lib/accessHelper';
+import { useEffect, useState } from 'react';
 
 export const LoginPage = () => {
+  const navigate = useNavigate();
+
+  // login page shouldn't be accessible for an authenticated user
+  useEffect(() => {
+    if (!accessExpired()) {
+      navigate('/');
+    }
+  });
+
   const validationSchema = Yup.object({
-    email: Yup.string().email('Invalid email address').required('Email is required').max(100, 'Email must be at most 100 characters'),
-    password: Yup.string()
-      .required('Password is required')
-      .min(6, 'Password must be at least 6 characters')
-      .max(255, 'Password must be at most 255 characters'),
+    email: Yup.string().email('Invalid email address').max(100, 'Email must be at most 100 characters'),
+    password: Yup.string().required('Password is required'),
   });
 
+  const [error, setError] = useState<string | null>(null);
+
   const handleSubmit = (values: LoginFormType) => {
     // Handle form submission here
-    console.log('Form values:', values);
+    authApi
+      .authControllerLogin(values)
+      .then((response) => {
+        if (response?.data?.accessToken) {
+          setAccessToken(response.data);
+          initializeApis();
+          navigate('/dashboard/profile');
```

Comment:
After submit, could you redirect the user in `/dashboard` instead of its profile?

Reply:
Ok